community/examples/ramble.yaml

```diff
@@ -13,11 +13,11 @@
 # limitations under the License.
 ---
 
-blueprint_name: ramble
+blueprint_name: ramble-01
```

Comment:
Ah... Sorry for confusion. Blueprint name should be `ramble` and deployment name should be `ramble-01`. 

Blueprint name should not change between deployments but deployment name should. This is mostly notional as users can do whatever they want but we are trying to set the tone. 

Please change blueprint name back and leave deployment name as you have it.

Reply:
Done